Sources/AblyAssetTrackingPublisher/WorkerQueue/PublisherWorkerSpecification.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public class PublisherWorkSpecification {
```

Comment:
Can we make this an `enum` instead so that in the worker factory we can avoid casting and also have a `switch` statement that checks for exhaustiveness?

Reply:
Yes! 9f7f37f